ros/src/tl_detector/tl_detector.py

```diff
@@ -81,27 +114,33 @@ def image_cb(self, msg):
         if self.state != state:
             self.state_count = 0
             self.state = state
-        elif self.state_count >= STATE_COUNT_THRESHOLD:
+        elif self.state_count >= self.statecount_threshold:
+	    if self.last_state != self.state:
```

Comment:
my eyes are hurting... ;-) there's a tab character instead of spaces before the `if self.last_state...`. PyCharm complains about mixed spaces and tabs for both line 117 and 118.

Reply:
pushed a fix now, all spaces here too.